Update README.md (#133)
nathanlesage/academics-on-mastodon: README.md

- [Curated Account Lists](#curated-account-lists) — find people to follow
- [Groups](#groups) — tap into what people are saying
- [Servers/Communities](#serverscommunities) — if you need to register for an account first, start here

> - [Additional Resources](#Additonal-Resources) — for guides and resources for navigating the platform

Reviewer: ```suggestion
- [Additional Resources](#additional-resources) — for guides and resources for navigating the platform
```

> - [Wider Fediverse Information](#Wider-Fediverse-Information) — for information and resources on other Fediverse applications

Reviewer: ```suggestion
- [Wider Fediverse Information](#wider-fediverse-information) — for information and resources on other Fediverse applications
```

- [Historians of Science, Medicine, and Technology](https://docs.google.com/spreadsheets/d/1MKivV_PkVZBAKBXpBkxNFEvjvYvUyE-DQHtujsh6Gww/) (Google Spreadsheet; [Form to add yourself](https://docs.google.com/forms/d/e/1FAIpQLSfBvVQv5TNW6s__ufEllGXiCP53zZuKLzYOoaaHFU7p7CiqKQ/viewform))
- [History and Philosophy of Science](https://oliviodare.github.io/mastodon-hps/)
- [History](https://docs.google.com/spreadsheets/d/1zmuH0ffMpCNOEjsReMk5iiqGwiUvhO2TtrM8QKu98SE/htmlview#gid=844879137) (Google Spreadsheet)

> - [Human-Computer Interaction](https://directory.hci.social/) - See also: [HCI Directory Bot](https://hci.social/@directory_bot)

Reviewer: Shouldn't we do this with other servers that have bots as well …? Just curious, we don't have to do that now!

Author: Sure, that is probably better!

- [baudigital.social](https://baudigital.social/) (Architecture, civil engineering, or urban studies; run by FID BAUdigital)
- [Biologists.social](https://biologists.social/) (The Company of Biologists community server: cell biology, developmental biology and experimental biology)
- [Climatescienc.es](https://climatescienc.es/explore) (Climate science and related fields)

> - [Clinicians Exchange](https://mastodon.clinicians-exchange.org/) - (For mental health professionals and the general public to exchange ideas, resources and expertise including but not limited to ethics, politics, practice building, and clinical issues; See [here](https://www.clinicians-exchange.org/) for more info.)

Reviewer: ```suggestion
- [Clinicians Exchange](https://mastodon.clinicians-exchange.org/) (For mental health professionals and the general public to exchange ideas, resources and expertise including, but not limited to, ethics, politics, practice building, and clinical issues; [more information](https://www.clinicians-exchange.org/))
```

Reviewer: Did you now take the suggestion, or only in parts, or not at all? GH is confusing me here …

[▲ Top](#academics-on-mastodon-)

## Additional Resources
- You can find [a comprehensive list of these and many more academic Mastodon servers on fediscience.org](https://fediscience.org/server-list.html). 
- Visit [fedi.tips](https://fedi.tips) for a general beginners guide to Mastodon. 
- [hcommons.org has created a quick start guide specifically for researchers in the humanities](https://hcommons.org/docs/mastodon-quick-start-guide-for-humanities-scholars/) that also includes a short list of instances that are tailored to humanities specifically.
- Visit the [Mastodon Media List](https://docs.google.com/document/d/1TTtXbf0yDXHKXwR9jpZ4kU79mmiZT5KIM7U4tBr83hY/edit#heading=h.m7f2jiwk8u11) for a comprehensive list of Academic Journals and News Outlets.
- [TrueSciPhi.org](https://truesciphi.org/) - Various comprehensive science, philosophy, and mathematics lists covering podcasts, Mastodon, and Twitter.

> - [OpenCheck.is](https://opencheck.is/people) - Additonal lists and a verification method for those migrating from Twitter.

Reviewer: ```suggestion
- [OpenCheck.is](https://opencheck.is/people) - Additional lists and a verification method for those migrating from Twitter.
```

- [StreetPass](https://streetpass.social/) - A browser extension that alerts you to Mastodon verification links on a Web site.

## Wider Fediverse Information

> ### General

Reviewer: ```suggestion

### General
```

- [fediverse.party](https://fediverse.party/en/miscellaneous/) - A listing of software on the Fediverse with descriptions.

> ### Lemmy/Kbin

Reviewer: ```suggestion
### Lemmy/Kbin

```

[Lemmy](https://join-lemmy.org/) and [Kbin](https://kbin.social/) are link aggregators on the Fediverse, similar to Reddit. They are generally compatible with each other.

- [sub.rehab](https://sub.rehab/) - A listing of where Reddit servers have migrated to, mostly Lemmy/Kbin.

> Servers:

Reviewer: ```suggestion
#### Servers

```

- [Clinicians Exchange](https://lem.clinicians-exchange.org/) - A Lemmy server for mental health professionals and the general public to exchange ideas related to mental health. See [here](https://www.clinicians-exchange.org/) for more info.
- [mander.xyz](https://mander.xyz/) - A Lemmy server dedicated to the sciences.

> ### Matrix

Reviewer: ```suggestion
### Matrix

```

[Matrix](https://matrix.org/) is a secure messaging application similar to Discord, Slack, Telegram, and WhatsApp.

- [Element](https://element.io/) - Popular mobile, web, and desktop application for using the Matrix network. 
- [Matrix](https://doc.matrix.tu-dresden.de/en/) guide from the University of Dresden.

> Servers:

Reviewer: ```suggestion
#### Servers

```

## Wider Fediverse Information

### General
- [fediverse.party](https://fediverse.party/en/miscellaneous/) - A listing of software on the Fediverse with descriptions.

### Lemmy/Kbin

[Lemmy](https://join-lemmy.org/) and [Kbin](https://kbin.social/) are link aggregators on the Fediverse, similar to Reddit. They are generally compatible with each other.

- [sub.rehab](https://sub.rehab/) - A listing of where Reddit servers have migrated to, mostly Lemmy/Kbin.

#### Servers

- [Clinicians Exchange](https://lem.clinicians-exchange.org/) - A Lemmy server for mental health professionals and the general public to exchange ideas related to mental health. See [here](https://www.clinicians-exchange.org/) for more info.
- [mander.xyz](https://mander.xyz/) - A Lemmy server dedicated to the sciences.

### Matrix

[Matrix](https://matrix.org/) is a secure messaging application similar to Discord, Slack, Telegram, and WhatsApp.

- [Element](https://element.io/) - Popular mobile, web, and desktop application for using the Matrix network. 
- [Matrix](https://doc.matrix.tu-dresden.de/en/) guide from the University of Dresden.

#### Servers

> - Archaeology: ```matrix.archaeo.social```

Reviewer: I don't quite remember how Matrix works; aren't there links…?

Author: No, they must sign up on a client application.
